code/modules/jobs/jobs.dm

```diff
@@ -59,6 +59,21 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 	"Cyborg",
 	"pAI"))
 
+GLOBAL_LIST_INIT(exp_jobsmap, list(

```

Comment:
Nope. The datums store the exp type needed to unlock them.
Note command jobs don't have an exp type of command. They have their relevant department.